src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -1711,6 +1711,45 @@ class C
                 Diagnostic(ErrorCode.ERR_MemberAlreadyExists, "F").WithArguments("F", "C").WithLocation(11, 17));
         }
 
+        [Fact]
+        public void Overloads_05()
+        {
+            var source =
```

Comment:
This test did make me wonder, do we have a test where `I1` defines a method with an `IntPtr` param, `I2` defines the same method with a `nint` parameter, and class `A` implements both interfaces implicitly?

Reply:
https://github.com/dotnet/roslyn/pull/42996